<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCalculator</class>
 <widget class="QWidget" name="ImageCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="image1ComboBox"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Image 2:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="operationComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Image 1:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="image2ComboBox"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QRadioButton" name="floatResRadioButton">
     <property name="text">
      <string>32-bit (float) result</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="calculateButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Operation:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QRadioButton" name="newWindowRadioButton">
     <property name="text">
      <string>Create new window</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
